### ISSUE TITLE
go though portal's inpage api

### ISSUE THREAD

**Issue by yqrashawn** [Conflux-Chain/conflux-portal#221]
there's a ongoing api migration in metamask right now
we need to go though our api see if we should follow metamask's change

https://docs.metamask.io/guide/ethereum-provider.html#legacy-methods

**Comment by Thegaram** [comment]
Somewhat related: It is confusing that, based on our [docs](https://developer.conflux-chain.org/docs/conflux-portal/docs/en/portal/API_Reference/provider_api#upcoming-provider-changes), these are already deprecated. I guess the reason is that our docs was mostly copied from MetaMask. (See `Upcoming Provider Changes`.)

**Comment by yqrashawn** [comment]
> Somewhat related: It is confusing that, based on our [docs](https://developer.conflux-chain.org/docs/conflux-portal/docs/en/portal/API_Reference/provider_api#upcoming-provider-changes), these are already deprecated. I guess the reason is that our docs was mostly copied from MetaMask. (See `Upcoming Provider Changes`.)

Yes. Both code and doc are forked from metamask and we removed the deprecation warning of those api in inpage.js. We need to go through the doc after our api change.
